helpers/addresses.py

```diff
@@ -239,6 +239,7 @@
         "bbveCVX-CVX-f": "0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551",
         "remBADGER": "0x6aF7377b5009d7d154F36FE9e235aE1DA27Aea22",
         "remDIGG": "0x99F39D495C6A5237f43602f3Ab5F49786E46c9B0",
+        "bcrvBadger": "0xeC1c717A3b02582A4Aa2275260C583095536b613"
```

Comment:
confirmed that is verified on etherscan, deployer is one of badger dao and `token` is `badgerWBTC_f`.

Reply:
+1